Clarify expectations and requirements for programs used for registrant filtering, seeding and automated refereeing in `Tournaments/Official_support` (#11465)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

**The tournament must also abide by the following practices throughout:**

- All registrants must be screened by the account support team before play commences.[^play-commence] See [tournament screening](#tournament-screening) for more details.

> - If a user satisfies the sign-up criteria (if any) and is not [filtered](#registrant-filtering-seeding-and-automated-refereeing) out through other means, the tournament must not prevent those who pass the screening from participating without both ample evidence presented publicly against them and the approval of the account support team.

Reviewer: only this section is relevant and not the umbrella section (that i suggested to be removed)

```suggestion
- If a user satisfies the sign-up criteria (if any) and is not [filtered](#registrant-filtering-and-seeding) out through other means, the tournament must not prevent those who pass the screening from participating without both ample evidence presented publicly against them and the approval of the account support team.
```

- An appropriately designed badge ([see below](#profile-badges)).
- A link to the publicly viewable catalogue of your tournament's match history.

> ### Registrant Filtering, Seeding and Automated Refereeing
> 

Reviewer: this looks weird a as a subsection. it's all under the `## Expectations` section anyway

```suggestion
```

> #### Expectations for Programs

Reviewer: ```suggestion
### Programs
```

Third-party programs designed to filter registrants, seed players, or automate the job of a referee must be open, transparent, and documented. Any third-party tool used for these purposes must:

- Be made publicly and functionally available as an open-source repository.

> - Be completely documented in an easily readable & digestable format.

Reviewer: ampersands aren't typically used in running text

```suggestion
- Be completely documented in an easily readable and digestible format.
```

Currently, approval for the use of any such tool must be **explicitly granted** by the osu! support team. Additionally, this approval must be granted for **every tournament where programs for registrant filtering, seeding, or automated refereeing are used.** We expect to relax some of these requirements in the future as the technology surrounding these programs develops and experience with using them grows.

> #### Registrant Filtering and Seeding

Reviewer: ```suggestion
### Registrant filtering and seeding
```

> Registrant filtering, not to be confused with [screening](#tournament-screening), is the process of using any program, algorithm, mathematical formula, or other objective means to remove otherwise good-standing players from a tournament. The standard use of such tools or systems is to reduce the variance in skill within a tournament. One such system that may come to mind is [BWS](/wiki/Tournaments/Badge-weighted_seeding).

Reviewer: can be expressed shorter

```suggestion
Registrant filtering, not to be confused with [screening](#tournament-screening), is the process of using any program, algorithm, mathematical formula, or other objective means to remove otherwise good-standing players from a tournament. This is commonly used reduce variance in skill within a tournament, e.g. with [BWS](/wiki/Tournaments/Badge-weighted_seeding).
```

Reviewer 2: think there's a missing "to" in the suggestion ("commonly used to")

In addition to the expectations for programs above, tools or systems used to filter registrants or seed players must also abide by the following practices:

- If a dataset is used, it must be up to date, publicly available and downloadable. Documentation must be provided explaining how to recreate the dataset and reproduce the results using it — including code where applicable.
- When doing qualifiers, the formula or method used to determine seeding must be included in the tournament's ruleset.

A third party should be able to get the same results without needing additional information. Manually filtering players or adjusting their seeds is strictly prohibited.

> #### Automated Refereeing

Reviewer: ```suggestion
### Automated refereeing
```

While support for advertising and sharing the outcome of your tournament is available via news posts, there are additional requirements that must be considered:

- Only news posts advertising registrations and/or recapping of **final** results are presently permitted. Additional news posts are presently not permitted.
  - Tournaments that are given permission to advertise registrations are also expected to provide a conclusionary news post recapping **final** results.

>   - If a news post advertising registrations is denied but all other [requirements](#requesting-in-game-banner-and-news-post-support) are met, a tournament may still be given permission for a conclusionary news post. Hosts and organisers interested in doing so should send in a new request after the tournament has concluded.

Reviewer: if a tournament is to have a news post at all, it makes no sense to wait until it has concluded most of the time because coverage is time-sensitive (depending on the content)
